acme/setup.py

```diff
@@ -35,6 +35,15 @@
 else:
     install_requires.append('mock')
 
+if sys.version_info < (3, 0):
```

Comment:
This kind of code makes me nervous. My understanding is that we'd have to start building two wheels of our packages, one for Python 2 and one for Python 3. It'd be nice to not have to do that if possible.

Looks like we're in luck though. Since version 1.12, `dnspython` is supports both Python 2 and Python 3. This version of `dnspython` seems to be available everywhere that we are packaged. With this in mind, I think we should change this to depend on `dnspython>=1.12`, allowing us to continue building a single wheel that should work everywhere. We should add a comment saying that version 1.12 is required to provide both Python 2 and Python 3 support.